src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ApprenticeshipIncentive.cs

```diff
@@ -496,6 +498,25 @@ public void ValidateMinimumRequiredAgreementVersion(Guid pendingPaymentId, Accou
             pendingPayment.AddValidationResult(PendingPaymentValidationResult.New(Guid.NewGuid(), collectionPeriod, ValidationStep.HasSignedMinVersion, isValid));
         }
 
+        public void ValidateEmploymentChecks(Guid pendingPaymentId, CollectionPeriod collectionPeriod)
+        {
+            var pendingPayment = GetPendingPaymentForValidationCheck(pendingPaymentId);
+
+            var employedAtStartOfApprenticeshipCheck = EmploymentChecks.FirstOrDefault(x =>
+                x.CheckType == EmploymentCheckType.EmployedAtStartOfApprenticeship);
+
+            var employedAtStartOfApprenticeshipResult = employedAtStartOfApprenticeshipCheck?.Result != null && employedAtStartOfApprenticeshipCheck.Result.Value;
+            
+            pendingPayment.AddValidationResult(PendingPaymentValidationResult.New(Guid.NewGuid(), collectionPeriod, ValidationStep.EmployedAtStartOfApprenticeship, employedAtStartOfApprenticeshipResult));
```

Comment:
Please put each check in it's own private method and call from here

Reply:
Updated